### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by camjc**
- Largest jump is Babel to v6.0.20.
- Babel's `export default` seems broken for now, will check back once v6 settles down.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,47 +26,50 @@
...
-    "chai": "^3.3.0",
+    "babel-loader": "^6.0.1",
+    "babel-plugin-transform-strict-mode": "^6.0.15",
+    "babel-preset-es2015": "^6.0.15",
+    "babel-preset-react": "^6.0.15",
+    "chai": "^3.4.0",
     "copyfiles": "^0.2.1",
-    "css-loader": "^0.20.1",
-    "eslint": "^1.7.2",
+    "css-loader": "^0.21.0",
```
Comment (mcslash) [inline package.json]: are we going back a version on purpose?

**Comment by omgaz** [comment]
LGTM :fish_cake:

File: package.json (hunk 1 above)
Comment (omgaz) [inline package.json]: 20 -> 21 looks like forward to me :P

Comment (mcslash) [inline package.json]: yes it is @omgaz, good observation.

**Comment by mcslash** [comment]
LGTJ.

**Comment by etaoins** [comment]
LGTM

**Comment by mcslash** [comment]
Merged! :rocket:

**Pull request merged**